### ISSUE TITLE
Fetch Balance of 'Claimable' recurring donations 

### ISSUE THREAD

**Issue by divine-comedian** [Giveth/impact-graph#1141]
From https://github.com/Giveth/giveth-dapps-v2/issues/3094

We need to fetch the balance of the super tokens for a given project's anchor address (allo protocol) (or recipient addresses for now). There will only be a limited amount of tokens we need to check for, those will be: 

- ETHx
- USDCx
- DAIx
- OPx
- GIVx (need to deploy)

We also need to show the dollar value for this token, as well as the combined dollar value for all balances

Here is the account API that supposedly should be able to do this for us: 
https://accounting.superfluid.dev/v1/swagger

**Comment by mohammadranjbarz** [comment]
@divine-comedian  Do we have any diagram ( I mean something like sequence diagram) to can understand easily the process? because it's a little confusing for me, we can create some diagrams together if you want after that it can be easier to talk about that or address issues to processes, for instance this is a diagram that we made for chainvine before


https://mermaid.live/edit#pako:eNp1Vslu2zAQ_ZWBLm0B9wd8CBDbbWE0aIy4cS6-0NRIYkyRKklZcIP-e2eozbGVIAeJnOW9N4v8lkibYjJPPP6p0UhcKZE7Ue4N0J-QwTp4wgydQ3dzhrg33WEdrKnLQ29UCReUVJUwAb47awKa9PZmIeRx8mJZCGVOyuDt1Q91Wgotay2CsmYrnarCrdVq0SPrwcPXu7sByhx2ChsPocDOQGjYOMvM4UGYVJkcNiJHoGeQWsmjh1eruph9mBhzwDqHxwqNB9kfMFYMBTyvISPJak8oggWHufKBnjl2jgadCAhamWMb3Vh6tScchZ_DfcYOOYbAyNh2xuDPIIUBXwiHHJlOlIPMKcLmJ-h3RZvDlj18-07Mx9ydxRSx9n038iJa12wcpsph6IlXzr6iDJ4S-Ai8YklJCl-hVJmSvQUHSq0hHVoYQ-qrqq1NLNnz00NHeqDXCK0xXAt4QWFZ-2BLdvWc7RqBh0ZpDQcEVVYaS6SMKRzOF9VkgkO-l5gPlIHaaQpo4fPLevPlVsUR_CoS9ATPleR9poykUd5KlWnbtBbU1RNd1o0KlY5r24pFlvQkgiAYUtexaVkeNyVKFyAGWy0ojjgRlj5ji2Vgt6Kgrdtq8T79402TDlctr5ZJBFmQCP0Yamsr2J6NLIiV-otDZg8vnPoXhsa6Iyzp-tUeWqdr3Att5TEWhOpZoDyO-H0QofYXboPpe_yDww4d1R_T6Uys0HOVxnqlH-W40ebndLCLLuTqxRoFRwZcMDxRq4HNJktBcj7QbAInr6VE77NaXxR_mPn7zXrMPdH_dD-nKNR0vIlOHfcx0uh86fIxvYny_y6UB_p3tJE4CRFzZzDYjAIK3Yhzlyvu_alpn1rwfdsvtzvmXPs4jA1tTStSUCEmlKOXnwo8blPa9CeVUmmHEaOg2ubKzOBEH4ZugoT23VZrhOsX6vRyGRIN24NryxJ6yzhJgk-BAlEBSAemUQpTU3m56T9e-4_0QabXdsNihDmmoi4qBCHkvhg7od3KMk5HB3wGGGQvdjJLSqRBVSl99N_4bJ-QZ4n7ZE6PKWai1mGf7M0_MuXvOk9tMg-uxllSx5nofiO8P_yWKvph0J79-w-cotnh

**Comment by divine-comedian** [comment]
## something like this? 

![image](https://github.com/Giveth/impact-graph/assets/67759413/50e279b3-7068-4b09-9c58-2d288cbadfd8)

**Comment by mohammadranjbarz** [comment]
Thanks so much, @divine-comedia , Now it's clearer for me. Do we have some test wallet addresses that have some balance or something to can play with APIs?

**Comment by divine-comedian** [comment]
Actually the superfluid app has testnet support - they can set you up with tokens and you can play around with sending tokens etc. 
https://app.superfluid.finance/

**Comment by divine-comedian** [comment]
@alireza-sharifpour 

I'm second guessing myself now on this issue - Do we need back-end services for this or can we query the subgraph directly to get claimable balances?

**Comment by jainkrati** [comment]
@divine-comedian so is this a Frontend tasks or it needs backend work as well/? Accordingly can we please assign to the right devs.

**Comment by divine-comedian** [comment]
> @divine-comedian so is this a Frontend tasks or it needs backend work as well/? Accordingly can we please assign to the right devs.


Yes! Thanks for the ping - We had a short call last week and we can fetch these values either using the subgraph or SDK I think - so no back-end work needed. 

I'm not an expert of which will be the most efficient for this case, assuming we might have to fetch the balance for multiple tokens on a single view.
